lib/TWCManager/Vehicle/TeslaAPI.py

```diff
@@ -351,6 +351,12 @@ def car_api_available(
                                 # reaching this point later.
                                 vehicle.delayNextWakeAttempt = 15 * 60
                         elif state == "offline":
+                            # In any case it can make sense to wait 10 seconds here.
+                            # I had the issue, that the next command was sent too
+                            # fast and only a reboot of the Raspberry resultet in
+                            # possible reconnect to the API (even the Tesla App
+                            # couldn't connect anymore).
+                            self.time.sleep(10)
```

Comment:
Would this impact the background tasks loop @MikeBishop? My recall of the Tesla API flow is quite poor but the existence of delays through other mechanisms makes me think that any delay should probably be more of a backoff timer than pausing execution

Reply:
Technically yes, but brief pauses aren't an issue.  There are other instances where the code will pause for a few seconds after certain commands finish before sending the next command.  So long as you're not substantially delaying the ability to send start/stop commands and not doing it over and over, it should be safe.

Reply:
I gave a try with 5 seconds. It works, too. So pushed another change here.